### PR TITLE
Add Sapper alias to Svelte

### LINKED ISSUE #3219: Sapper icon
**Name:** Sapper
**Website:** https://sapper.svelte.dev/
**Official resources for icon and color:**
  
The same Svelte icon but with `#159794` brand color

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
Sapper

**Issue:** Closes #3219
**Alexa rank:** n/a
**GitHub stars:** [~7.1k](https://github.com/sveltejs/sapper)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [ ] I optimized the icon with SVGO or SVGOMG
  - [ ] The SVG `viewbox` is `0 0 24 24`

### Description
Colour from [SVG](https://sapper.svelte.dev/sapper-logo-horizontal.svg) in [source](https://sapper.svelte.dev/) header.

### CODE REVIEW

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @PeterShaggyNoble! As it's confirming #3270 I'll wait for another approval on this before merging.

**Comment by PeterShaggyNoble** [comment]
Putting this on hold temporarily pending one last (hopefully shorter!) internal discussion on an alternative way of handling duplicate icons.

**Review by service-paradis — APPROVED** [review]
LGTM 💯

**Pull request merged**
